package.json

```diff
@@ -52,7 +52,9 @@
     "clsx": "^1.2.1",
     "dayjs": "^1.11.11",
     "hls.js": "^1.5.11",
-    "solid-js": "^1.8.17"
+    "mapbox-gl": "^3.5.1",
+    "solid-js": "^1.8.17",
+    "solid-map-gl": "^1.11.3"
```

Comment:
why do we need two new dependencies? what are they for? how big are they?

Reply:
mapbox-gl is for the map itself. The static map on the cards is actually an image retrieved from the mapbox api and can't be interacted with like a normal map. solid-map-gl is a solid plugin that is essentially a wrapper around mapbox-gl that helps keep the code concise. mapbox-gl can be used standalone, but the diff would be too large. As for how large they are, mapbox-gl is a bit large (more here https://github.com/mapbox/mapbox-gl-js) and solid-map-gl is small (https://github.com/GIShub4/solid-map-gl). If you're okay with the larger diff, solid-map-gl can be removed, but mapbox-gl can't. However, if you're okay with the map not being interactive, I might be able to do it with zero additional deps.